### PR TITLE
Website main and about

### PR DESCRIPTION
**Pull request by yara-58**
Still needs some styling

### CODE REVIEW

File: website/src/components/About.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                registry versions to create a robust application.
+            </li>
+            
+        </ul>
+    </div>
+</template>
+
+<script>
+export default {
+  name: 'Home'
```
Comment (yara-58) [inline website/src/components/About.vue]: change to 'About'

Comment (szczura) [reply website/src/components/About.vue]: Can we change this before approving it and merging it?

Comment (szczura) [reply website/src/components/About.vue]: Can we change this before approving it and merging it?

**Comment by szczura** [comment]
Will resolve #199

**Comment by szczura** [comment]
Can we fix this and keep the header straight:
image

Also either left-align the first header or center the second header to keep it consistent:
image

**Comment by yara-58** [comment]
aboutpage
homepage

thats how they look when i run it locally

**Comment by szczura** [comment]
> thats how they look when i run it locally

I'm not sure why the bullet points are aligned differently, but there's still the issue with the header/navigation bar. "About" and "Developer Team" are a different color and aligned differently compared to the other links.

File: website/src/routes.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 import Login from './components/LoginForm.vue';
 
+
 const root = '/SeeShells/'
 
 const routes = [
     { path: root, component: Home },
     { path: root + 'register', component: Register },
+    { path: root + 'team', component: TeamMembers },
+    { path: root + 'about', component: About}
```
Comment (szczura) [inline website/src/routes.js]: There needs to be a comma after this line

File: website/src/components/About.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<template>
+    <div id="about">
+        <h1>About Our Project</h1>
+        <p align="left">
+            <br />
+            SeeShells is essentially an information extract software. The objective is to
```
Comment (bridCquinn) [inline website/src/components/About.vue]: I would say that the "essentially" word kind of looks unprofessional.

File: website/src/components/About.vue [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            create a standalone open source executable that can run both online and offline.
+            It extracts and parses through Windows Registry information. This data This data
```
Comment (klaki892) [inline website/src/components/About.vue]:
```suggestion
            It extracts and parses through Windows Registry information. This data 
```

**Pull request merged**